api_app/analyzers_manager/file_analyzers/zippy_scan.py

```diff
@@ -17,31 +17,36 @@ class ZippyAnalyser(FileAnalyzer):
     Tells if a file is written by HUMAN or AI
     """
 
+    ENGINES = {
+        "lzma": CompressionEngine.LZMA,
+        "zlib": CompressionEngine.ZLIB,
+        "brotli": CompressionEngine.BROTLI,
+    }
+
     engine: str
 
     def update(self):
         pass
 
     def run(self):
-        z = None
-        if self.engine == "lzma":
-            z = Zippy(engine=CompressionEngine.LZMA)
-        elif self.engine == "zlib":
-            z = Zippy(engine=CompressionEngine.ZLIB)
-        elif self.engine == "brotli":
-            z = Zippy(engine=CompressionEngine.BROTLI)
-        else:
-            z = EnsembledZippy()
+        z = (
+            Zippy(engine=self.ENGINES[self.engine])
+            if self.engine in self.ENGINES
+            else EnsembledZippy()
+        )
 
         logger.info("Running Zippy on file %s using %s", self.filepath, self.engine)
```

Comment:
may I ask you to use f-strings instead of %s?

Reply:
noted